assets/training/aoai/proxy_components/src/finetune_submit.py

```diff
@@ -83,8 +99,6 @@ def submit_finetune_job(self, training_file_id, validation_file_id, model, regis
         """metrics can be retrived only for successful finetune job"""
         finetuned_model_id = finetune_job.fine_tuned_model
         logger.info(f'Fine-tune job: {self.job_id} finished successfully. model id: {finetuned_model_id}')
-        logger.info("fetching training metrics from Azure OpenAI")
```

Comment:
I think we should emit here also, to not miss any metrics

Reply:
calling one more time after status becomes terminal